From 38fa80e38cf348dc599a9f01e5a2cd052e5af775 Mon Sep 17 00:00:00 2001
From: Steve Sakoman <steve@sakoman.com>
Date: Sat, 8 Oct 2011 16:17:42 -0700
Subject: [PATCH 209/214] omap: hsmmc: Increase dto to deal with timeout
 errors

A number of SD cards are still experiencing timeout errors.  This
can lead to data corruption in some cases.  This patch increases
the dto value even further.
---
 drivers/mmc/host/omap_hsmmc.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/mmc/host/omap_hsmmc.c b/drivers/mmc/host/omap_hsmmc.c
index 5dfb0f1..6910aa3 100644
--- a/drivers/mmc/host/omap_hsmmc.c
+++ b/drivers/mmc/host/omap_hsmmc.c
@@ -1424,7 +1424,7 @@ static void set_data_timeout(struct omap_hsmmc_host *host,
 	cycle_ns = 1000000000 / (clk_get_rate(host->fclk) / clkd);
 	timeout = timeout_ns / cycle_ns;
 	timeout += timeout_clks;
-	timeout *= 2;
+	timeout *= 3;
 	if (timeout) {
 		while ((timeout & 0x80000000) == 0) {
 			dto += 1;
-- 
1.7.5.4

